pkg/registry/apis/provisioning/github.go

```diff
@@ -107,6 +108,24 @@ func (r *githubRepository) Read(ctx context.Context, filePath string, commit str
 	return []byte(data), nil
 }
 
+func (r *githubRepository) Write(ctx context.Context, path string, data []byte, comment string) error {
+	return &errors.StatusError{
+		ErrStatus: v1.Status{
+			Message: "write file is not yet implemented",
+			Code:    http.StatusNotImplemented,
+		},
+	}
+}
```

Comment:
can we also get a `TODO` for handling symlinks properly (maybe that should exist in the endpoint impl, though...)? i.e. we don't want to allow a symlink to e.g. `/run/secrets/xyz.json` (or any config files, or whatever else may exist) to ever be read and written elsewhere, for example. it's easier to just bar all symlinks, hence just rejecting them outright would work very well.

important because git supports symlinks to arbitrary paths, even if they don't exist in the same repo